tests/test_emitter.py

```diff
@@ -49,7 +52,7 @@
         WindowsApiEmitter as Emitter
     )
 elif platform.is_bsd():
-    from watchdog.observers.kqueue import (
+    from watchdog.observers.kqueue import (  # type: ignore[attr-defined,no-redef]
```

Comment:
Do we need to keep around the mypy-incompatible platform checks?  Or can we apply the compatible ones everywhere?  I think the result in terms of ignoring this code is the same unless we have BSD systems in the test matrix, but just as a general thing I wonder about fully dropping the incompatible check mechanism.  Even if there is agreement on that, I understand that that could well end up as a breaking change to save for a major version bump.  Though, it does appear to be not documented which makes it at least a little less clearly public API.  `:]`  https://watchdog.readthedocs.io/en/latest/api.html#module-watchdog.utils